Create executive-director.md (#1318)
USRSE/usrse.github.io: pages/about/executive-director.md

---
layout: page
title: Executive Director
permalink: /about/executive-director/
subtitle: Sustainability for the US-RSE Association 
menubar: about
menubar_toc: true
set_last_modified: true
---

### Executive Director

Sandra Gesing
[sandra\@us-rse.org](mailto:sandra@us-rse.org)

### Message from the Executive Director October 2023

Dear US-RSE members and allies,
 
I’m honored and thrilled to be appointed the inaugural Executive Director of US-RSE. My whole work life has been 

> immersed around software with me starting already at high-school with programming. I enjoyed working in industry as 

Reviewer: ```suggestion
immersed around software with me starting with programming in high school. I enjoyed working in industry as 
```

an administrator, system developer and head of a systems programming group for over a decade and then I switched 

> from industry to research completing my PhD in a bioinformatics group. I have been an RSE, research faculty and 

Reviewer: ```suggestion
from industry to research, completing my PhD in a bioinformatics group. I have been an RSE, research faculty and 
```

senior research scientist with a major focus on research software in the last 17 years. This moment is very special to me 
because it is about a community I’m incredibly passionate about, and this community feels like home. Being a part 
of it means having a space for discussions, a space for learning and support of a non-traditional career path. 
With my resume, I often felt like a unicorn – not within US-RSE though.

The last five years have shown how much we all can achieve together as a community, and we went from around 20 members 

> in 2018 to nearly 2000 in 2023. US-RSE is a vibrant community evident in the activities on Slack, working groups, 

Reviewer: ```suggestion
in 2018 to over 2000 in 2023. US-RSE is a vibrant community, as evident in the activities on Slack, working groups, 
```
